### ISSUE TITLE
About some given synthesized images

### ISSUE THREAD

**Issue by JianrongYan** [PeterWang512/AttributeByUnlearning#2]
Hi, thanks for your open source excellent work first. 
I am curious about  the details of 110 synthesized images provided by you. Is they generated using the weight named model.bin under the model_fisher folder?  And for reproducibility, could you please provide more implementation details when generating it, e.g., random seed, scheduler.

**Comment by PeterWang512** [comment]
Yes, they are generated by the model. I will also release code for generating the images in the future. Thanks!

**Comment by PeterWang512** [comment]
Sorry for the delay. The code has been updated to do this. Thanks!

**Comment by enkeejunior1** [comment]
I was also having difficulty with reproduction, but I think I've discovered the cause.

> **Main Q**: Is the SD only customized by textual inversion? I think it is Dreambooth, which includes textual inversion + model weight finetuning. 

Long version: After downloading using download.sh and examining the data, I found that besides the special token embedding vector `<new>.bin`, there are also finetuned KV weights `pytorch_custom_diffusion_weights` included. If you try to generate images using only `<new>.bin`, it fails to generate customized samples. However, I was able to reproduce customized samples after loading the KV from `pytorch_custom_diffusion_weights.pt`. 

So, I'm wondering if each image wasn't customized with textual inversion, but rather with something like dreambooth that includes KV.
